### DISCUSSION TITLE
阿里云盘下载文件超时

### DISCUSSION THREAD

**Discussion by qinians** [AlistGo/alist#437] [Q&A]
阿里云盘下载文件超时

**Comment by xhofe** [comment]
Further information is requested

**Reply by wu529778790** [reply]
> Further information is requested

下载一个118g的文件，下载到一半就没速度了，试了几次都是这样，开不开代理都这样，比如下载这个https://alist.shenzjd.com/%E9%98%BF%E9%87%8C%E4%BA%91%E7%9B%98/%E5%AD%A6%E4%B9%A0%F0%9F%90%9F/%E6%8B%89%E5%8B%BE-%E5%A4%A7%E5%89%8D%E7%AB%AF%E9%AB%98%E8%96%AA%E8%AE%AD%E7%BB%83%E8%90%A5%20%7C%20%E5%AF%B9%E6%A0%87%E9%98%BF%E9%87%8Cp7

**Reply by xhofe** [reply]
阿里的下载链接有有效期，与本程序无关。
